kolibri/plugins/coach/assets/src/views/common/QuizLessonDetailsHeader.vue

```diff
@@ -27,6 +27,16 @@
       </div>
     </div>
     <MissingResourceAlert v-if="resource.missing_resource" />
+    <UiAlert
+      v-if="isFromOldKolibiri && hideAlert"
+      type="warning"
+      class="old-kolibri-banner"
+      @dismiss="hideAlert = false"
```

Comment:
I'm finding the `hideAlert` here a little bit confusing. When I see that variable name, I would assume that it means that the UiAlert is not visible. However, since `isFromOldKolibiri && hideAlert` would have to be true for this to render, it means that `hideAlert` would be true for this to show.  And, the `@dismiss` setting `hideAlert` to false. 

I think having this be `showAlert` would be clearer.